assets/sass/components/dashboard/_googlesitekit-ReportMetric.scss

```
@@ -77,6 +77,12 @@
 	vertical-align: bottom;
 }
 
+.googlesitekit-pagespeed-report-metric__hint-text {
+	color: $c-neutral-n-500;
+	letter-spacing: 0.25px;
+	margin-top: 7px;
```

Let's use a relative 0.5em value as that appears to align more with the AC (half of 1em).

```suggestion
	margin-top: 7px;
```